distro/wildfly/compatibility-subsystem27/pom.xml

```
@@ -0,0 +1,196 @@
+              <resources>
+                <resource>
+                  <directory>${basedir}/../subsystem/src/main/java</directory>
+                  <filtering>false</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-main-resources</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/generated-resources/subsystem</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${basedir}/../subsystem/src/main/resources</directory>
+                  <filtering>false</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <!-- copy test sources & resources -->
+          <execution>
+            <id>copy-test-sources</id>
+            <phase>generate-test-sources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/generated-test-sources/subsystem</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${basedir}/../subsystem/src/test/java</directory>
+                  <filtering>false</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-test-resources</id>
+            <phase>generate-test-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/generated-test-resources/subsystem</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${basedir}/../subsystem/src/test/resources</directory>
+                  <filtering>false</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>add-subsystem-source</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>add-source</goal>
+            </goals>
+            <configuration>
+              <sources>
+                <source>${project.build.directory}/generated-sources/subsystem</source>
+              </sources>
+            </configuration>
+          </execution>
+          <execution>
+            <id>add-jakarta-resource</id>
```

🔧 Copy-paste issue? 🙂 
```suggestion
            <id>add-subsystem-resources</id>
```